src/Graphics/Image/Processing/Deconvolution.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE BangPatterns #-} 
+
+module Graphics.Image.Processing.Deconvolution where
+
+import Graphics.Image as I
+import Graphics.Image.Interface as I
+import Prelude as P
+
+-- | Supplementary function needed to blur an image and test the algorithm.
+randomBlur :: Image VS X Double
```

Comment:
Anything that is needed for testing, suppose to live in the test suite

Reply:
@lehins Okay sir, so this should be in the `usage` section in the documentation?

Reply:
Yes you could move it to function documentation as an example. In that case I'd also recommend collapsing the big example (note `====__Examples__`):

```
--  ====__Examples__
--
-- <<images/frog.jpg>>   <<images/frog_deconvolve.jpg>>
-- 
--  Usage:
-- >>> u0 <- readImage' "images/frog.jpg" :: IO (Image VS RGB Double)
-- ...
```

In order to check out the affect, run `stack haddock --open`

Reply:
@lehins Okay sir, will change it.